src/Apps/Ingress/Pages/IngressApp/IngressApp.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useCallback, useReducer } from 'react';
+import {
```

Comment:
You'll wanna make sure you add an `index.ts` file for exporting the entry points